Formula/bazarr.rb

```diff
@@ -0,0 +1,43 @@
+class Bazarr < Formula
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  url "https://github.com/morpheus65535/bazarr/releases/download/v1.0.3/bazarr.zip"
+  sha256 "540a5c4d80110ba35a566c17829cd6f31444db7ade7c6b9d0603eccaaed8a628"
+  license "GPL-3.0-only"
+  head "https://github.com/morpheus65535/bazarr.git", branch: "master"
+
+  depends_on "gcc"
+  depends_on "numpy"
+  depends_on "python@3.9"
+
+  uses_from_macos "zlib"
+
+  def install
+    libexec.install Dir["*"]
+    (libexec/"data").install_symlink pkgetc => "config"
+
+    (bin/"bazarr").write_env_script Formula["python@3.9"].opt_bin/"python3", "#{libexec}/bazarr.py", []
```

Comment:
If you don't actually need to set env variables then you should probably use `write_exec_script`.

Reply:
Okay, cheers.

Reply:
Ah, the issue is that I need to specify args still. So looks like this is necessary?